ReplaceWords.java

```diff
@@ -0,0 +1,85 @@
+/**
+Time Complexity : O(height of the trie)
+Space Complexity : O(N*M)
+LeetCode Submitted : Yes
```

Comment:
time complexity will be mp+mk where m is no.of elements and p and k are sizes of the words